js/kucoin.js

```diff
@@ -799,7 +801,7 @@ module.exports = class kucoin extends Exchange {
         let network = this.safeStringUpper (params, 'network');
```

Comment:
@AlgoQ here I was suggesting using `safeStringUpper2` and accept both `chain` and `network`